lte/gateway/c/session_manager/SessionStateEnforcer.cpp

```diff
@@ -96,6 +96,7 @@ bool SessionStateEnforcer::m5g_init_session_credit(
     const std::string& session_id, const SessionConfig& cfg) {
   /* creating SessionState object with state CREATING
    * This calls constructor and allocates memory*/
+  // TODO(veshkemburu): Figure out how to handle sharding(object)
   auto session_state =
       std::make_unique<SessionState>(imsi, session_id, cfg, *rule_store_);

```

Comment:
are you missing a 0 in the end there?

Reply:
(placeholder for 0 shard)

Reply:
No need for a 0 here because I can just initialize shard id to be 0 within the class itself. It's a different constructor